src/wp-includes/ms-network.php

```diff
@@ -107,35 +108,44 @@ function clean_network_cache( $ids ) {
  * cache using the network group with the key using the ID of the networks.
  *
  * @since 4.6.0
+ * @since 6.1.0 Introduced the `$update_meta_cache` parameter.
  *
- * @param array $networks Array of network row objects.
+ * @param array $networks          Array of network row objects.
+ * @param bool  $update_meta_cache Whether to update sitemeta cache. Default true.
```

Comment:
Should us `site meta` instead of `sitemeta` as per the existing [document here](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/ms-site.php#L368)
```suggestion
 * @param bool  $update_meta_cache Whether to update site meta cache. Default true.
```